### PR TITLE
System boot datasource migration to TPF

### PR DESCRIPTION
**Pull request by prajwalpatil25**
image

### CODE REVIEW

**Comment by rounak-adhikary** [comment]
Please enable linting of this datasource

File: redfish/provider/data_source_redfish_system_boot.go [hunk 1]
```diff
@@ -51,7 +206,7 @@ package provider
 // 		"resource_id": {
 // 			Type:        schema.TypeString,
```
Comment (rounak-adhikary) [inline redfish/provider/data_source_redfish_system_boot.go:207]: Please remove these comments too. The rest is good.

Comment (prajwalpatil25) [reply redfish/provider/data_source_redfish_system_boot.go]: done

**Review by rounak-adhikary — APPROVED** [review]
LGTM

**Review by Krishnan-Priyanshu — APPROVED** [review]

**Pull request merged**